selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/GitCompareTest.java

```diff
@@ -65,181 +94,152 @@
-        TestMenuCommandsConstants.Project.New.NEW,
-        TestMenuCommandsConstants.Project.New.JAVA_CLASS);
-    loader.waitOnClosed();
-    askForValueDialog.waitNewJavaClassOpen();
-    askForValueDialog.typeTextInFieldName("A");
-    askForValueDialog.clickOkBtnNewJavaClass();
-    askForValueDialog.waitNewJavaClassClose();
-    projectExplorer.waitItem(PROJECT_NAME + "/src/main/java/com/codenvy/example/spring/A.java");
-    projectExplorer.waitAndSelectItem(PROJECT_NAME);
-    menu.runCommand(TestMenuCommandsConstants.Git.GIT, TestMenuCommandsConstants.Git.ADD_TO_INDEX);
-    git.waitAddToIndexFormToOpen();
-    git.confirmAddToIndexForm();
-    git.waitGitStatusBarWithMess(TestGitConstants.GIT_ADD_TO_INDEX_SUCCESS);
-    projectExplorer.waitAndSelectItem(PROJECT_NAME);
-    menu.runCommand(
-        TestMenuCommandsConstants.Git.GIT,
-        TestMenuCommandsConstants.Git.Compare.COMPARE_TOP,
-        TestMenuCommandsConstants.Git.Compare.COMPARE_LATEST_VER);
-    loader.waitOnClosed();
+    // check compare after adding and deleting files
+    createNewJavaFile();
+    deleteJavaFile();
+    addToIndex();
+
+    menu.runCommand(GIT, COMPARE_TOP, COMPARE_LATEST_VER);
     git.waitGroupGitCompareIsOpen();
     git.waitExpTextInGroupGitCompare(TEXT_GROUP);
-    git.closeGroupGitCompareForm();
-    projectExplorer.waitAndSelectItem(
-        PROJECT_NAME + "/src/main/java/com/codenvy/example/spring/HelloWorld.java");
-    menu.runCommand(TestMenuCommandsConstants.Edit.EDIT, TestMenuCommandsConstants.Edit.DELETE);
-    loader.waitOnClosed();
-    acceptDialogWithText("Delete file \"HelloWorld.java\"?");
-    loader.waitOnClosed();
-    projectExplorer.waitAndSelectItem(PROJECT_NAME);
-    menu.runCommand(
-        TestMenuCommandsConstants.Git.GIT,
-        TestMenuCommandsConstants.Git.Compare.COMPARE_TOP,
-        TestMenuCommandsConstants.Git.Compare.COMPARE_LATEST_VER);
-    loader.waitOnClosed();
+    git.selectFileInChangedFilesTreePanel("Aclass.java");
+    git.clickOnGroupCompareButton();
+    git.waitGitCompareFormIsOpen();
+    gitCompare.waitTextNotPresentIntoLeftEditor("public class Aclass");
+    gitCompare.waitExpectedTextIntoRightEditor("public class Aclass");
+    git.closeGitCompareForm();
+
     git.waitGroupGitCompareIsOpen();
-    git.waitExpTextInGroupGitCompare(TEXT_GROUP);
-    git.selectFileInChangedFilesTreePanel("A.java");
+    git.selectFileInChangedFilesTreePanel("NewClass.java");
     git.clickOnGroupCompareButton();
     git.waitGitCompareFormIsOpen();
-    git.waitExpTextIntoCompareLeftEditor("public class A");
-    git.setFocusOnLeftGitCompareEditor();
-    git.setCursorToLine(2, LEFT_COMPARE_ST);
-    git.typeTextIntoGitCompareEditor("//***che***codenvy***");
-    git.waitExpTextIntoCompareLeftEditor("//***che***codenvy***");
-    git.clickOnGitCompareCloseButton();
-    askDialog.confirmAndWaitClosed();
-    git.waitGitCompareFormIsClosed();
+    git.waitExpTextIntoCompareLeftEditor("public class NewClass");
```

Comment:
Could you please add comment into the code with description of steps? For example - to explain where we are checking compare after adding, and where - after deleting.